### ISSUE TITLE
Reconfix-v0.1.0-fork

### ISSUE THREAD

**Issue by mbalamat** [resin-io/reconfix#60]
[Here](https://github.com/balena-io/reconfix/tree/reconfix-v0.1.0-fork) is the fork of v0.1.0 that works with Node14.

**Comment by mbalamat** [comment]
Connects-to: https://github.com/balena-io/balena-cli/issues/2079

**Comment by Ereski** [comment]
@mbalamat can you use `reconfix-v0.1.0-fork` for the CLI? This branch is a dead end and eventually I'll replace the version used by the CLI with #59

**Comment by mbalamat** [comment]
@Ereski yes I'll use https://github.com/balena-io/reconfix/tree/reconfix-v0.1.0-fork on all the modules that depend on it. It's a temporary solution until you finish with the new reconfix, feel free to update all the modules that depend on reconfix when the time comes. For now, I just don't want anyone to delete this branch, because I am installing it from GitHub via npm. That's why I mainly created this issue.
